src/tests/ftest/container/snapshot_functionality_test.yaml

```diff
@@ -0,0 +1,31 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+    test_machines:
+        - boro-A
+        - boro-B
+timeout: 100
```

Comment:
Can you please suggest if this timeout is at least twice the total run time of this test?